### PR TITLE
Feature/koala pickup hint

### PR DESCRIPTION
**Pull request by latedude2**

### CODE REVIEW

File: Assets/Scripts/HoldObject.cs [hunk 1]
```diff
@@ -92,20 +98,18 @@ private GameObject HighlightPickupable()
         {
             if (hit.collider.gameObject.CompareTag("PickableObject"))
             {
+                if(pickedUpObject == null)
+                    pickupHint.SetActive(true);
+                if(pickedUpObject != null)
+                    pickupHint.SetActive(false);
```
Comment (Jokubas126) [inline Assets/Scripts/HoldObject.cs]:
```suggestion
                    pickupHint.SetActive(pickedUpObject == null);
```

Comment (Jokubas126) [reply Assets/Scripts/HoldObject.cs]: Just a small improvement

**Review by Jokubas126 — APPROVED** [review]
Just add this little improvement and it's fine 👌

File: Assets/Scripts/HoldObject.cs (hunk 1 above)
Comment (latedude2) [reply Assets/Scripts/HoldObject.cs]: smart

**Pull request merged**